data/json/monstergroups/zombie_upgrades.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "type": "monstergroup",
+    "name": "GROUP_SOLDIER_UPGRADE",
+    "default": "mon_zombie_soldier_kevlar_1",
```

Comment:
This id doesn't exist.  You either failed to commit it, or it's supposed to be mon_zombie_soldier.

Reply:
I thought I caught all of those. Better make sure my repo is up to date